server/server.js

```diff
@@ -0,0 +1,26 @@
+const express = require('express')
```

Comment:
Across files, you using single quotes or double quotes, please standardize it. Preferable single quotes